src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.heartbeat
+
+import akka.actor._
+import scala.concurrent.duration._
+
+// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
```

Comment:
I see. No big deal. We don't generate any docs so far, so that's that.

In general, the way to go would be to
```
  /**
    * [docs here]
    */
```

(IntelliJ happens to generate annotations for arguments/return type and such if you type the `/**` and hit enter fwiw.)